specs/mediawiki/v1/content.yaml

```diff
@@ -75,7 +70,7 @@ paths:
         # See also http://tools.ietf.org/html/rfc7231#section-7.1.2
         uri: /{domain}/sys/parsoid/html/{title}/latest
 
-  /{module:page}/{title}/html/:
+  /{module:page}/html/{title}/:
```

Comment:
@earldouglas, exactly. It's about representations of the same resource, not normally different resources.